bluepyopt/ephys/locations.py

```diff
@@ -392,6 +393,35 @@ def instantiate(self, sim=None, icell=None):
         return self.find_icomp(sim, branches)
 
 
+class NrnTrunkSomaDistanceCompLocation(NrnSecSomaDistanceCompLocation):
+    """Location at a distance from soma along a main direction.
+
+    This is most useful to follow the trunk of an apical dendrite
+    without knowing the apical point, but only that apical trunk goes along y.
+    """
+
+    def set_sec_index(self, icell=None, direction=[0.0, 1.0, 0.0]):
+        """Search for the point furthest away along given direction."""
```

Comment:
Could you add a little doc about the direction argument.

Reply:
I added more doc!